worker-sandbox/src/lib.rs

```diff
@@ -21,6 +23,34 @@ mod r2;
 mod test;
 mod utils;
 
+#[cfg(feature = "http")]
```

Comment:
These just convert the types between what fetch expects (depends on feature flag) and what the test harness router expects (`worker::Request`). I think pretty soon we should have the `http` version of the test harness use `axum` and remove this.

Reply:
If the only reason you see for that API is to be used by the tests, then they should be dropped.

Regarding tests I'm bit more opinionated, as I believe we should move to wasm-bindgen-test and move unit tests closer to the source.

As MrBBot no longer told me to wait, so I started getting my feet wet...
https://github.com/rustwasm/wasm-bindgen/issues/3891

So most of the tests would be direct tests instead of the current model.